packages/inter-protocol/test/vaultFactory/test-liquidation-plans.js

```diff
@@ -0,0 +1,99 @@
+import { test as unknownTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+
+import { makeTracer } from '@agoric/internal';
+import { SECONDS_PER_DAY } from '../../src/proposals/econ-behaviors.js';
+import {
+  makeAuctioneerDriver,
+  makeDriverContext,
+  makeManagerDriver,
+} from './driver.js';
+
+/**
+ * @typedef {import('./driver.js').DriverContext & {
+ * }} Context
+ */
+/** @type {import('ava').TestFn<Context>} */
+const test = unknownTest;
+
+const trace = makeTracer('TestLP');
```

Comment:
```suggestion
const trace = makeTracer('TestLP', false);
```

Even if we want to turn on trace in many parts of the production code, I think there's no benefit to leaving it on in tests.

Reply:
done